packages/sdks/src/blocks/video/video.lite.tsx

```diff
@@ -47,6 +48,7 @@ export default function Video(props: VideoProps) {
   return (
     <video
       {...state.spreadProps}
+      preload={props.preload}
```

Comment:
```suggestion
      preload={props.preload || 'metadata'}
```

Default values on inputs do not apply retroactively, meaning that content that is already published will have this as undefined

Reply:
Oh right ! Thanks. So Ill also add it to the Video.tsx file in react sdk